### PR TITLE
tests: new test for snap logs command

### PR DESCRIPTION
**Pull request by sergiocazzolato**

### CODE REVIEW

File: tests/main/snap-logs/task.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    snap logs test-snapd-service.test-snapd-other-service | MATCH "running"
+
+    echo "check output lines for the logs"
+    snap logs -n=1 test-snapd-service | MATCH "running"
+    snap logs -n=all test-snapd-service | MATCH "running"
+    
+    echo "check -f option works"
+    snap logs -f test-snapd-service > service.log &    
+    snap stop test-snapd-service
+    pkill -f "snap logs -f test-snapd-service"
```
Comment (bboozzoo) [inline tests/main/snap-logs/task.yaml]: You could also do `kill $!` which should kill the snap logs command that was put to run in the background

**Comment by sergiocazzolato** [comment]
Currently the tests are failing because an issue on ubuntu-14.04-64

**Comment by sergiocazzolato** [comment]
This is the error which we see on trusty, 
google:ubuntu-14.04-64 .../tests/main/snap-logs# journalctl -n all
Failed to add match 'all': Invalid argument

**Comment by sergiocazzolato** [comment]
Fix for current error on: https://github.com/snapcore/snapd/pull/5591

**Pull request merged**